cmd/dockerd/builtins/genwinres_windows.go

```diff
@@ -1,5 +1,5 @@
-//go:generate go-winres make --arch=386,amd64,arm,arm64 --in=../../cli/winresources/dockerd/winres.json --out=../../cli/winresources/dockerd/resource
+//go:generate go-winres make --arch=386,amd64,arm,arm64 --in=../../../cli/winresources/dockerd/winres.json --out=../../../cli/winresources/dockerd/resource
```

Comment:
Reminds me we still need to move the remaining bits related to this, but I didn't manage to get that working yet; got confused by the various paths used;

- https://github.com/moby/moby/issues/44641
- https://github.com/moby/moby/pull/46886

Reply:
I don't have much idea exactly how this works, but this change should still work